src/media/js/app.js

```diff
@@ -51,6 +52,7 @@ const reducer = combineReducers({
   addonReview,
   addonSubmit,
   addonSubmitVersion,
+  addonThread,
```

Comment:
This file is getting a little big, could we maybe put `reducer` into its own file?